services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_scheduler.py

```diff
@@ -328,7 +325,11 @@ async def mark_service_for_removal(
                 )
                 return
 
-            current.dynamic_sidecar.service_removal_state.mark_to_remove(can_save)
+            # PC-> ANE: could you please review what to do when can_save=None
+            assert can_save is not None  # nosec
```

Comment:
Can save, should not longer be None. Please open an issue for this one.

Reply:
created issue